### PR TITLE
docs(ui events page): CFY-4407

### PR DESCRIPTION
**Pull request by edenpessach**

### CODE REVIEW

File: content/manager_webui/logs-events-page.md [hunk 1]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
 weight: 140
 
 terminology_link: reference-terminology.html
 ---
 {{% gsSummary %}}{{% /gsSummary %}}
 
-# Overview
-When clicking on the `Logs & Events` tab, you will be able to view the last 50 events aggregated from all `Blueprints` and `Deployments`:<br/>
-![Logs&EventsPage]({{< img "ui/ui-logsNevents-overview.png" >}})
+When clicking on the `Logs & Events` tab, you are presented with 50 events rows aggregated from all `Blueprints` and `Deployments` ordered latest first.<br/>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: events row -> event rows
ordered latest first -> ordered by timestamp (descending)

File: content/manager_webui/logs-events-page.md [hunk 2]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+![logs & events page]({{< img "ui/eventsPage/eventsPage.png" >}})<br><br>
+You can control which logs & events are displayed by:
 
+# Filtering
+### Blueprints
+To filter logs & events by specific blueprints choose one or more blueprints, matching logs & events will be automatically updated.<br/>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: blueprints, -> blueprints for which

File: content/manager_webui/logs-events-page.md [hunk 3]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+![blueprints filter]({{< img "ui/eventsPage/blueprintsFilter.png" >}})<br>
 
-You can control which events are displayed by filtering by:
+### Deployments
+To filter logs & events by specific deployments choose one or more deployments, matching logs & events will be automatically updated.<br/>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: same

File: content/manager_webui/logs-events-page.md [hunk 4]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+![deployments filter]({{< img "ui/eventsPage/deploymentsFilter.png" >}})<br>
 
-# Blueprints
-See the definition [here]({{page.terminology_link}}#blueprint).<br/>
-![Blueprints]({{< img "ui/ui-logsNevents-blueprint-selection.png" >}})
+### Log Levels
+To filter logs & events by specific log levels choose one or more log levels, matching logs & events will be automatically updated.<br/>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: same

File: content/manager_webui/logs-events-page.md [hunk 5]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+![log levels filter]({{< img "ui/eventsPage/loglevelsFilter.png" >}})<br>
+
+### Event Types
+To filter logs & events by specific event types choose one or more event types, matching logs & events will be automatically updated.<br/>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: same

File: content/manager_webui/logs-events-page.md [hunk 6]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+![event types filter]({{< img "ui/eventsPage/eventTypesFilter.png" >}})<br>
+
+### Time Range
+To filter logs & events from and to a specific timestamp use the time range filters.
+The filters consist of 2 timepickers, each allow manually typing the timestamp expected to be in `YYYY-MM-DD HH:MM` format or choosing a date and time from the timepicker.<br>
+The first timepicker filter logs & events `from` the chosen timestamp, the second filter log & events `to` the chosen timestamp.
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: filter -> filters (twice)
timestamp, -> timestamp while

File: content/manager_webui/logs-events-page.md [hunk 7]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+You can choose to filter using either of the timepickers or both.
+![time range filters]({{< img "ui/eventsPage/timeRangeFilter.png" >}})<br>
+
+
+{{% gsNote title="Note" %}}
+If it is unclear is your time is valid, look for this <i class="fa fa-calendar"></i> icon. When it shows - the input is valid and the data is automatically changed to match the filter.<br>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: is your time -> if your timestamp

File: content/manager_webui/logs-events-page.md [hunk 8]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+![time range]({{< img "ui/eventsPage/timeRange.png" >}})<br>
+{{% /gsNote %}}
+
+### Message Text
+To filter logs & events by specific message text free type the message, matching logs & events will be automatically updated.<br/>
+![message text filter]({{< img "ui/eventsPage/messageTextFilter.png" >}})<br>
+
+### Clear Filters
+To clear all of the filters you can easily do so by clicking the 'Clear Filters' button.
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: you can click the 'Clear Filters' button.

File: content/manager_webui/logs-events-page.md [hunk 9]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+![time range filters]({{< img "ui/eventsPage/timeRangeFilter.png" >}})<br>
+
+
+{{% gsNote title="Note" %}}
+If it is unclear is your time is valid, look for this <i class="fa fa-calendar"></i> icon. When it shows - the input is valid and the data is automatically changed to match the filter.<br>
+![time range]({{< img "ui/eventsPage/timeRange.png" >}})<br>
+{{% /gsNote %}}
+
+### Message Text
+To filter logs & events by specific message text free type the message, matching logs & events will be automatically updated.<br/>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: by a specific message's text you can free type the text and matching...

File: content/manager_webui/logs-events-page.md [hunk 10]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+![message text filter]({{< img "ui/eventsPage/messageTextFilter.png" >}})<br>
+
+### Clear Filters
+To clear all of the filters you can easily do so by clicking the 'Clear Filters' button.
+
+# Sorting
+To sort logs & events click on the desired table header to switch order between ascending, descending and none.
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: table -> table's
order -> the ordering

File: content/manager_webui/logs-events-page.md [hunk 11]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+# The Log/Event
+By default the table is not presenting all of the log/event information. In order to reveal all of the log/event information simply click on the row itself.<br>
+An event example:
+![event example]({{< img "ui/eventsPage/event.png" >}})<br>
+
+A log example:
+![log example]({{< img "ui/eventsPage/log.png" >}})<br>
+
+{{% gsNote title="Note" %}}
+Not all logs/events has the same fields and there are some fields which are specific to a type.<br>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: has -> have
a type .. please explain. A type of what?

File: content/manager_webui/logs-events-page.md [hunk 12]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+For example `Logger` and `Log Level` fields are specific to logs and `Event Type` field is specific to events.
+{{% /gsNote %}}
+
+# Adjust The Table
+By default the logs & events table is presenting all of the columns, you can choose which columns you would like the table to present.
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: columns, -> columns but

File: content/manager_webui/logs-events-page.md [hunk 13]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+Click on the <i class="fa fa-cog"></i> icon, a dropdown menu containing all of the table headers names is presented.<br>
+![choose columns]({{< img "ui/eventsPage/columnsOrganizer.png" >}})<br>
+
+Clicking on a header toggles if it should be presented, additionally you can drag and drop headers to determine the order of the columns.<br>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: presented. Additionally..

File: content/manager_webui/logs-events-page.md [hunk 14]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+Here is an example of a customised table:
+![customized table]({{< img "ui/eventsPage/customizedTable.png" >}})<br>
+
+# Persistency
+Filtering, sorting are paging persistent. This means that while you filter, sort and changing pages all of those preference automatically change the url of the page.<br>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: Can you please try and explain this better. I'm not sure a user will understand what it means.

File: content/manager_webui/logs-events-page.md [hunk 15]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
...
+This for example allow you to refresh the page and not loose all the preference, bookmark it, or send the link to a colleague.
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: allow -> allows
loose -> lose
all the preference -> all preferences
bookmark it -> bookmark the page

File: content/manager_webui/logs-events-page.md [hunk 16]
```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
 title: The Logs & Events Page
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: this page is wonderful!

File: content/manager_webui/logs-events-page.md (hunk 11 above)
Comment (edenpessach) [inline content/manager_webui/logs-events-page.md]: regarding type- meant type as log/event.
As even you were confused about it , it is obviously not good enough for docs.. so rephrased to this:
"Not all logs/events have the same fields as there are some fields which are specific to logs and others which are specific to events."

File: content/manager_webui/logs-events-page.md (hunk 14 above)
Comment (edenpessach) [inline content/manager_webui/logs-events-page.md]: Done. Please review the new explanation before merging.

File: content/manager_webui/logs-events-page.md (hunk 15 above)
Comment (edenpessach) [inline content/manager_webui/logs-events-page.md]: allow -> allows
loose -> lose
all the preference -> all preferences 
===> Was removed when rephrasing the explanation
bookmark it -> bookmark the page
===> Is fixed.

File: content/manager_webui/logs-events-page.md (hunk 16 above)
Comment (edenpessach) [inline content/manager_webui/logs-events-page.md]: Thanks a lot! :smile:

**Comment by edenpessach** [comment]
Please make sure to check all comments before merging!
I am solving conflicts and pushing now.

File: content/manager_webui/logs-events-page.md [hunk 17]
```diff
@@ -3,38 +3,90 @@ layout: bt_wiki
...
+To filter logs & events by specific event types choose one or more event types for which matching logs & events will be automatically updated.<br/>
+![event types filter]({{< img "ui/eventsPage/eventTypesFilter.png" >}})<br>
 
-# Timeframe
-A list of timeframes. This selection will affect how far back in history the events will be shown.<br/>
-You can choose a timeframe as short as 5 minutes or as long as 5 days, with increments in between in the minute, hour and day range.<br/>
-![Timeframe]({{< img "ui/ui-logsNevents-timeframe-selection.png" >}})
+### Time Range
+To filter logs & events from and to a specific timestamp use the time range filters.
+The filters consist of 2 timepickers, each allow manually typing the timestamp expected to be in `YYYY-MM-DD HH:MM` format or choosing a date and time from the timepicker.<br>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: consists

File: content/manager_webui/logs-events-page.md [hunk 18]
```diff
@@ -3,38 +3,90 @@ layout: bt_wiki
...
+# Sorting
+To sort logs & events click on the desired table's header to switch the ordering between ascending, descending and none.
+![sorting]({{< img "ui/eventsPage/sorting.png" >}})<br>
+
+# Pagination
+50 logs & events are presented per page. You can use the pagination bar to scroll through the pages.
+![pagination]({{< img "ui/eventsPage/pagination.png" >}})<br>
+
+# The Log/Event
+By default the table is not presenting all of the log/event information. In order to reveal all of the log/event information simply click on the row itself.<br>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: is not presenting -> does not present

File: content/manager_webui/logs-events-page.md [hunk 19]
```diff
@@ -3,38 +3,90 @@ layout: bt_wiki
...
+An event example:
+![event example]({{< img "ui/eventsPage/event.png" >}})<br>
+
+A log example:
+![log example]({{< img "ui/eventsPage/log.png" >}})<br>
+
+{{% gsNote title="Note" %}}
+Not all logs/events have the same fields as there are some fields which are specific to logs and others which are specific to events.<br>
+For example `Logger` and `Log Level` fields are specific to logs and `Event Type` field is specific to events.
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: logs and -> logs while the

File: content/manager_webui/logs-events-page.md [hunk 20]
```diff
@@ -3,38 +3,90 @@ layout: bt_wiki
...
+{{% /gsNote %}}
+
+# Adjust The Table
+By default the logs & events table is presenting all of the columns but you can choose which columns you would like the table to present.
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: is presenting -> presents

File: content/manager_webui/logs-events-page.md [hunk 21]
```diff
@@ -3,38 +3,90 @@ layout: bt_wiki
...
+Click on the <i class="fa fa-cog"></i> icon, a dropdown menu containing all of the table headers names is presented.<br>
+![choose columns]({{< img "ui/eventsPage/columnsOrganizer.png" >}})<br>
+
+Clicking on a header toggles if it should be presented. Additionally you can drag and drop headers to determine the ordering of the columns.<br>
+Here is an example of a customised table:
+![customized table]({{< img "ui/eventsPage/customizedTable.png" >}})<br>
+
+# Persistency
+While you adjust the table's view by filtering, sorting and paging through it, those preferences are automatically saved at the page's url.<br>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: saved at -> saved as part of
I think that sounds a tad more accurate.

File: content/manager_webui/logs-events-page.md [hunk 22]
```diff
@@ -3,38 +3,90 @@ layout: bt_wiki
...
+For every unique set of table filtering, sorting and paging preferences there is a unique url to match it and when loading the page those preferences will be automatically loaded from the page's url, by thus adjusting the table's view.
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: by thus -> thus

File: content/manager_webui/logs-events-page.md [hunk 23]
```diff
@@ -3,38 +3,90 @@ layout: bt_wiki
...
+This for example allow you to refresh the page, bookmark the page, or send the current page's url as a link to a colleague without having to set the view's preferences all over again when loading the page.
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: allows

refresh the page, bookmark the page, -> refresh or bookmark the page or send..

File: content/manager_webui/logs-events-page.md [hunk 24]
```diff
@@ -3,38 +3,90 @@ layout: bt_wiki
...
+
+{{% gsNote title="Note" %}}
+The `table's view` is the set of filtering, sorting and paging preferences applied on the table. The data for a specific view is dynamic and is not considered as part of the view's preferences.<br>
```
Comment (nir0s) [inline content/manager_webui/logs-events-page.md]: applied on -> applied to
considered as part -> considered part

**Comment by nir0s** [comment]
please merge after additional fixes

**Pull request merged**